compiler/reader_test.go

```diff
@@ -71,6 +71,7 @@ func (s *ReaderTestingSuite) TestDisableInfoCache(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(petstore, NotNil)
 	c.Assert(len(infoCache), Equals, 0)
+	EnableInfoCache()
```

Comment:
Caches are disabled at the start of this (and the following) test. In an order-dependence, subsequent tests failed because they expected the caches to be enabled.